docs/_guide/styles.md

```diff
@@ -4,66 +4,247 @@ title: Styles
 slug: styles
 ---
 
```

Comment:
I'm wanting an intro section with a more general name. Like:

Overview

LitElement provides flexibility in how you style your components, and ensures interoperability through style encapsulation using Shadow DOM, the browser's built-in way of encapsulating native UI components.

Why use Shadow DOM? Shadow DOM scopes CSS so that...